packages/components/tabs/src/tab-group.spec.ts

```diff
@@ -11,10 +11,18 @@ describe('sl-tab-group', () => {
     beforeEach(async () => {
       el = await fixture(html`<sl-tab-group></sl-tab-group>`);
     });
-  
+
     it('should not break', () => {
       expect(el).shadowDom.to.equalSnapshot();
     });
+
+    it('should have horizontal layout by default', () => {
+      expect(el.getAttribute('vertical')).to.equal('false');
```

Comment:
`expect(el).not.to.have.attribute('vertical');` (as a result of the earlier `@property` comment)